test/integration/generate-roms.js

```diff
@@ -7,13 +7,13 @@ const fs = require('fs'),
     romDir = path.join(__dirname, 'test-roms'),
     RomCommands = require('./rom-commands'),
     spawnAndWait = require('../../util/spawn-and-wait'),
-    bin = "../../../dist/create-nes-game" + (os.platform() === 'linux' ? '-linux' : '');
```

Comment:
I'm pretty sure nodejs is smart enough to hide this from us, but if we want to do this, I'd request that we use the `path` module to normalize it, rather than including the path twice.